PayPalPayoutsSdk/Payouts/PayoutItemResponse.cs

```diff
@@ -33,12 +33,6 @@ public class PayoutItemResponse
         [DataMember(Name="currency_conversion", EmitDefaultValue = false)]
         public PayoutCurrencyConversion CurrencyConversion;
 
-        /// <summary>
-        /// The error details.
-        /// </summary>
-        [DataMember(Name="errors", EmitDefaultValue = false)]
-        public PayoutError Errors;
```

Comment:
this has to be replaced with Error to represent the item failure details

Reply:
done